amida-tech/indaba-client
# Bug/inba 814 jf survey back to draft

#### What does this PR do?
Extracts the Survey and Project Status modals into redux-forms. **Is dependent on INBA-811.**

#### Related JIRA tickets:
INBA-811.
INBA-814.
Also, the issue of INBA-844 (https://jira.amida-tech.com/browse/INBA-844).

#### How should this be manually tested?
Go to a project and use the Project and Survey Status modals. Ensure that all behavior is consistent as was with before. Note the problem with INBA-844 (trying to start a project when the survey is in draft). This is a new bug unrelated to this issue. 

#### Background/Context
In truth, the initial problem should be taken care of with INBA-811. However, I seized upon this as an excuse to refactor the Status modals. 

#### Screenshots (if appropriate):
